kyo-core/shared/src/main/scala/kyo/Channel.scala

```diff
@@ -248,7 +248,7 @@ object Channel:
                 Loop[Unit, Unit, Abort[Closed] & Async & Emit[Chunk[A]]](()): _ =>
                     Channel.take(self).map: a =>
                         drainEffect.map: chunk =>
-                            Emit.valueWith(Chunk(a).concat(chunk))(Loop.continue)
+                            Emit.valueWith(Chunk.single(a).concat(chunk))(Loop.continue)
```

Comment:
I've opened https://github.com/getkyo/kyo/pull/1161 so we don't need to explicitly call `Chunk.single`